src/Action/Api/BaseApiAwareAction.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Marem\PayumPaybox\Action\Api;
+
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\ApiAwareInterface;
+use Payum\Core\Exception\UnsupportedApiException;
+use Marem\PayumPaybox\Api;
+
+abstract class BaseApiAwareAction implements ActionInterface, ApiAwareInterface
```

Comment:
there is a trait for it https://github.com/Payum/Payum/blob/master/src/Payum/Core/ApiAwareTrait.php

could be used like this: https://github.com/Payum/Payum/blob/5009b89a92b0977cd1d6fc93426bbd3ec108ba5a/src/Payum/Paypal/Masspay/Nvp/Action/Api/MasspayAction.php#L22